Components/Button/button.css

```diff
@@ -0,0 +1,80 @@
+@import url("/Components/Alert/alert.css");
+.btn {
+  width: 12rem;
+  height: 5rem;
+  margin: 5px;
+  border-radius: 5px;
+  font-size: large;
+}
+
+.solid-btn {
+  color: white;
+  border: none;
+}
+
+.primary-hover:hover {
+  background-color: var(--primary-hover);
+  cursor: pointer;
+}
+
+.secondary-hover:hover {
+  background-color: var(--secondary-hover);
+  cursor: pointer;
+}
+
+.success-hover:hover {
+  background-color: var(--success-hover);
+  cursor: pointer;
+}
+
+.warning-hover:hover {
+  background-color: var(--warning-hover);
+  cursor: pointer;
+}
+
+.outline-primarybtn:hover {
+  background-color: var(--primary-hover);
+  color: var(--background-color);
+  cursor: pointer;
+}
+
+.outline-secondarybtn:hover {
+  background-color: var(--secondary-hover);
+  color: var(--background-color);
+  cursor: pointer;
+}
+
+.outline-successbtn:hover {
+  background-color: var(--success-hover);
+  color: var(--background-color);
+  cursor: pointer;
+}
+
+.outline-warningbtn:hover {
+  background-color: var(--warning-hover);
+  color: var(--background-color);
+  cursor: pointer;
+}
+
+.linkbtn:hover {
+  width: 12.5rem;
+  height: 5.5rem;
```

Comment:
Because of the hover effect, the UI is getting distorted. Maybe you can add some other effects for hovering. Other than this, its good.